assembly/src/parsers/field_ops.rs

```diff
@@ -4,6 +4,7 @@ use super::{
     Node::{self, Instruction},
     ParsingError, StarkField, Token,
 };
+use vm_core::{Felt, FieldElement};
```

Comment:
Let's import `ONE` from crate here. Then, on line 24 you can just do `if imm == ONE {`.